tests/test_telemetry.py

```diff
@@ -52,15 +51,19 @@ def test_emit_usage_metrics_fails(mock_httpx_get, caplog):
         "python_version": "3.11",
         "platform_system": "darwin",
         "platform_machine": "amd64",
-        "variables": {"a": 1, "b": 2},
+        "event_type": "dag_run",
+        "status": "success",
+        "dag_hash": "d151d1fa2f03270ea116cc7494f2c591",
+        "task_count": 3,
     }
     is_success = telemetry.emit_usage_metrics(sample_metrics)
     mock_httpx_get.assert_called_once_with(
-        "https://astronomer.gateway.scarf.sh/dag-factory/v1/0.2.0a1/2.10.1/3.11/darwin/amd64?dagfactory_version=0.2.0a1&airflow_version=2.10.1&python_version=3.11&platform_system=darwin&platform_machine=amd64&variables=%7B%27a%27%3A+1%2C+%27b%27%3A+2%7D",
+        "https://astronomer.gateway.scarf.sh/v1/0.2.0a1/2.10.1/3.11/darwin/amd64/dag_run/success/d151d1fa2f03270ea116cc7494f2c591/3",
```

Comment:
Why is the URL missing `/dag-factory/`?

Since Astronomer is building our own dashboards, and very few people have access to Scarf right now, wouldn't it be better to continue passing most arguments as query strings - so we can easily extend them, if needed?

Reply:
> Why is the URL missing /dag-factory/?

Maybe I'll have to add that in path 

<img width="1065" alt="Screenshot 2024-11-29 at 10 36 28 PM" src="https://github.com/user-attachments/assets/cdff267b-abfc-4799-836e-db7183e6711b">

Reply:
Fixed in c9f9b7104871ae619c19d2e68c97f344ee963ce7